src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiEndpoint: 'https://localhost:44378/api/v1'
+  apiEndpoint: 'https://localhost:44378/api/v1',
```

Comment:
Haven't we decided to use port 5000 for the API?

Reply:
@wizacass if it's going to be https, it has to be 5001

Reply:
Done

Reply:
Lets leave it as http for now, as we might have problems with self-signed certificates on localhost. And once we do production release, then lets switch to https.